docs/lib/storage/fragments/android/upload.md

```diff
@@ -136,3 +137,60 @@ upload
 
 </amplify-block>
 </amplify-block-switcher>
+
+## Upload inputStream
```

Comment:
Let's put this one _above_ `uploadFile`, as it is the more commonly used one on Android.  iOS is also similar - they have `uploadData` (equivalent of `uploadInputStream`) above `uploadFile`